protocols/client/client.go

```diff
@@ -407,12 +409,20 @@ func HybridVSockDialer(sock string, timeout time.Duration) (net.Conn, error) {
 			return nil, err
 		}
 
-		// Read EOT (End of transmission) byte
-		eot := make([]byte, 32)
-		if _, err = conn.Read(eot); err != nil {
-			// Just close the connection, gRPC will dial again
-			// without errors
-			conn.Close()
+		// Receive the package from the connection without remove this
```

Comment:
Nit: s/package/packet/.

Just an idea but you could simplify this comment to something like:

> Receive the packet from the connection without removing it from the receive queue (MSG_PEEK), ensuring the connection is usable.

Reply:
@jodh-intel Thanks!  Pushed a new version according to your comments.